### DISCUSSION TITLE
please create a discord server

### DISCUSSION THREAD

**Discussion by pmnzt** [nuxt-modules/i18n#3050] [Ideas]
I am encountering insane amount of bugs, i have been engaged in the issue tracker and opened several issue in the past, but it is really difficult for me to open an issue every single time i find a bug, and the worst part is, those bugs aren't even in big project, just fresh projects and i already have issue with nuxt-i18n. We need to have a discord server, where we could help each other.

Not going to lie, but this is the only module i have faced too many bugs from the first day i used, to this day.
Sorry if this feels like i am not appreciating your efforts, i really do, but i just couldn't cope with what's happening.

**Comment by BobbieGoede** [comment]
I'm not sure how a Discord server would make it easier for you to report bugs, we would still need to ask the same questions (minimal reproduction, steps to reproduce, etc.) while also fragmenting issue tracking.

I suggest opening issues for each bug you run into and/or opening pull requests to fix these, you have opened 4 issues in the past, in what way is it difficult for you to open a new issue?

**Reply by pmnzt** [reply]
Creating a discord server is still a good idea, but anyway.

here is an issue i have just opened https://github.com/nuxt-modules/i18n/issues/3051
